pandas/io/gbq.py

```diff
@@ -477,11 +477,17 @@ def verify_schema(self, dataset_id, table_id, schema):
         from apiclient.errors import HttpError
 
         try:
-            return (self.service.tables().get(
+            original_schema = self.service.tables().get(
                 projectId=self.project_id,
                 datasetId=dataset_id,
                 tableId=table_id
-            ).execute()['schema']) == schema
+            ).execute()['schema']
+
+            modified_schema_field_list = [
+                {key: field[key] for key in field if key != 'description'}
```

Comment:
Could we add a unit test to confirm that `'description'` is ignored when comparing to the schema of an existing table?